benchmarks/views/model.py

```diff
@@ -37,9 +38,10 @@ def view(request, id: int):
 
 
 def determine_context(id, request):
+
     # this is a bit hacky: we're loading scores for *all* public models as well as *all* of the user's models
     # so we're loading a lot of unnecessary detail. But it lets us re-use already existing code.
-    reference_context = get_context(None)  # public models
+    reference_context = get_context(show_public=True)
```

Comment:
this isn't right: if you include _all_ models here, including the non-public ones, then the ranking in the model view is different from the main site.
Maybe we need to add a message to competition-models' view that the ranks are with respect to the main site and not the competition leaderboard. (Or we rank competition models with respect to the competition only)